keps/672-execution-policy/README.md

```diff
@@ -0,0 +1,401 @@
+          - launcher
+        replicatedJobsStatus: Ready
+  replicatedJobs:
+    - name: initializer
+      template:
+        spec:
+          template:
+            spec:
+              containers:
+                - name: model-initializer
+                  image: docker.io/kubeflow/model-initializer
+                  volumeMounts:
+                    - mountPath: /workspace/pre-trained-model
+                      name: model-initializer
+                - name: dataset-initializer
+                  image: docker.io/kubeflow/dataset-initializer
+                  volumeMounts:
+                    - mountPath: /workspace/dataset
+                      name: dataset-initializer
+              volumes:
+                - name: dataset-initializer
+                  persistentVolumeClaim:
+                    claimName: dataset-initializer
+                - name: model-initializer
+                  persistentVolumeClaim:
+                    claimName: model-initializer
+    - name: launcher
+      template:
+        spec:
+          parallelism: 1
+          completions: 1
+          template:
+            spec:
+              containers:
+                - name: launcher
+                  image: docker.io/kubeflow/mpi-launcher
+                  resources:
+                    limits:
+                      nvidia.com/gpu: 5
+                  volumeMounts:
+                    - mountPath: /workspace/dataset
+                      name: dataset-initializer
+                    - mountPath: /workspace/pre-trained-model
+                      name: model-initializer
+              volumes:
+                - name: dataset-initializer
+                  persistentVolumeClaim:
+                    claimName: dataset-initializer
+                - name: model-initializer
+                  persistentVolumeClaim:
+                    claimName: model-initializer
+    - name: trainer-node
+      template:
+        spec:
+          parallelism: 3
+          completions: 3
+          template:
+            spec:
+              containers:
+                - name: trainer
+                  image: docker.io/kubeflow/deepspeed-trainer
+                  resources:
+                    limits:
+                      nvidia.com/gpu: 5
+                  volumeMounts:
+                    - mountPath: /workspace/dataset
+                      name: dataset-initializer
+                    - mountPath: /workspace/pre-trained-model
+                      name: model-initializer
+              volumes:
+                - name: dataset-initializer
+                  persistentVolumeClaim:
+                    claimName: dataset-initializer
+                - name: model-initializer
+                  persistentVolumeClaim:
+                    claimName: model-initializer
+```
+
+#### Story 3
+
+As an HPC user, I want to run a set of simulations with MPI that complete successfully
+(or some percentage) before using the data for a next step analysis.
```

Comment:
Some percentage is out of scope for this..

Reply:
Good point, let me remove it.